presets/aura/autocomplete/index.js

```diff
@@ -28,11 +28,11 @@ export default {
 
 			// Flex
 			'flex items-center flex-wrap',
-			'gap-2',
+			'gap-x-2',
 
 			// Spacing
 			'm-0 list-none',
-			'py-1 px-1',
+			'px-3',
```

Comment:
```suggestion
			'px-1',
```

Now that it has lower y padding, the `x-3` value doesn't actually look good.